traitsui/qt4/tabular_editor.py

```diff
@@ -517,19 +517,31 @@ def _selected_column_changed(self, selected_column):
     def _scroll_to_row_changed(self, row):
         """ Scroll to the given row.
         """
-        scroll_hint = self.scroll_to_row_hint_map.get(
-            self.factory.scroll_to_row_hint, self.control.PositionAtCenter
-        )
+        if self.factory.scroll_to_position_hint:
+            scroll_hint = self.scroll_to_position_hint_map.get(
+                self.factory.scroll_to_position_hint,
+                self.control.EnsureVisible
```

Comment:
this line is kinda unnecessary because of the if statement...
Nonetheless, this is what the default should be, so this is sort of being redundantly explicit